README.md

```diff
@@ -0,0 +1,52 @@
+# Final Project - *Earthquake App*
+
+**Earthquake App** is an android app that allows a user to view real time earthquake information. The user could request info by location, time, magnitude. It will alert user when a earthquake happens nearby.
+Users can notify their friend also they can mark themselves safe after an earthquake happens.
+
+Time spent: **X** hours spent in total
+
+## User Stories
+
+The following functionality is completed:
+
+* [ ] User can **sign in using other account such as Facebook, Twitter**
+* [ ] User can **view earthquakes on a map**
+* [ ] User can **filter earthquake by location, time and magnitude**
+* [ ] User can get **alert when an earthquake happen**
+  * [ ] User can set **time, location and magnitude of an earthquake to be alerted**
+* [ ] User can **share about an earthquake happening on social media facebook/twitter**
+* [ ] User can **make themself safe or send SOS info**
+
+
```

Comment:
What do we do about adding friends and getting the "safe" info about them ? Optional?
What do you think?